lib/dolly/query.rb

```diff
@@ -18,7 +18,10 @@ def find *keys
         if keys.count > 1
           build_collection( query_hash )
         else
-          self.new.from_json( database.all_docs(query_hash).parsed_response )
+          doc = self.new
+          response = database.all_docs(query_hash).parsed_response
```

Comment:
well is not this one, is that the parsed response now is a pased Hash and not a Json string like it used to be